docs/guides/dkms.md

```diff
@@ -101,22 +101,30 @@ MODULES="apple-bce"
 And then run `sudo mkinitcpio -P`.
 See your distro-specific instructions for configuring `apple-bce` to added to your initramfs.
 
-# Module configuration
+# Setting up the Touch Bar
 
-The Touchbar module offers some modes to set. In `/etc/modprobe.d/apple-tb.conf`, set `fnmode` (`options apple-ib-tb fnmode=x`) to one of the following options:
+The Touch Bar can be set up by running [this script](../tools/touchbar.sh) **in Linux** using `bash /path/to/script`. Make sure your macOS is updated to the latest version before running this script.
 
-- 0: Only show F1-F12
-- 1: Show media and brightness controls, use the `fn` key to switch to F1-12
-- 2: Show F1-F12, use the `fn` key to switch to media and brightness controls
-- 3: Only show media and brightness controls
-- 4: Only show the escape key
+After running this script, restart your Mac. The Touch Bar should be able to change modes by pressing the fn key.
+
+If you wish to change the default mode of the Touch Bar, run `sudo touchbar` and choose the mode you wish.
+
+!!! info "Keyboard Backlight"
+    The Keyboard Backlight controls on some models may break on changing the default mode of the touchbar. You may fix them by simply rebooting your Mac.
```

Comment:
Do you know which models? Is this caused by reloading the module so the sysfs entry is recreated and gnome etc doesn't use the new sysfs entry (so editing apple::kbd_backlight/brightness manually would change the brightness?)

Reply:
I guess the ones having magic keyboard backlight.

Reply:
I actually am doubtful for other Macs as well

Reply:
they have the keyboard backlight through hid-apple which doesn't get unloaded

Reply:
Oh this line is removed now.